run_bot.sh

```
@@ -590,6 +590,151 @@ test_recording() {
+        print_error "bot.config.json not found!"
+        print_info "Please create bot.config.json with your meeting configuration"
+        return 1
+    fi
+    
+    # Build image if necessary
+    if ! docker images | grep -q meet-teams-bot; then
+        print_info "Docker image not found, building..."
+        build_image
+    fi
+    
+    local output_dir=$(create_output_dir)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)